src/components/Sidebar/SidebarContent.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+// import { storeCustomers } from "../data";
+import { CustomerConsumer } from "../CustomerContext";
+
+export default function SidebarContent() {
+  return (
+    <div className="side-menu-container">
+      <CustomerConsumer>
+        {(ctx) => {
+          return (
+            <>
+              <select
+                id="storeCustomers"
+                onChange={(e) => ctx.setCustomer(e.target.value)}
+                value={(Object(ctx.customer) || {}).id || "DEFAULT"}
```

Comment:
what's the point of wrapping with Object() ?

Reply:
My mistake, something wasn't working, but now it works.